tvb/recon/cli/mrtrix.py

```diff
@@ -165,11 +167,34 @@ class tck2connectome(BaseMtrixCLI):
     """
 
     class Flags(BaseMtrixCLI.Flags):
-        assignment_radial_search = "-assignment_radial_search"
-        assignment_end_voxels = "-assignment_end_voxels"
+        out_assignments = "-out_assignments"
         scale_length = "-scale_length"
         stat_edge = "-stat_edge"
 
+    class Assignment(metaclass=abc.ABCMeta):
```

Comment:
`<jealousy>` Rust does this well; it's enum values can have associated data ([cf](https://doc.rust-lang.org/book/enums.html)). I wish they'd done that in Python's enum.`</jealousy>`

I might have tried just some classmethods
```python
class Assignment:

    def __init__(self, suffix, *extras):
        self.flag = '-assignment_' + suffix
        self.extras = extras

    def to_args(self):
        return [self.flag] + list(self.extras)

    @classmethod
    def end_voxels(self):
        return cls(suffix='end_voxels')

    @classmethod
    def radial_search(self, radius):
        return cls(suffix='end_voxels', str(radius))
```
wdyt?